README.md

```diff
@@ -17,18 +17,27 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
- SNMP traps that are generated by devices can be recieved by fluentd listener. The traps recieved by fluentd are formatted into a json output.                                                               
-                                                                                                                                                             
- @type ===> type of input plugin (required) ===> 'snmptrapalert'                                                                                             
- tag ===> tag that needs to be attached in front of each alert recieved ==> default value is "SNMPTrap.Alert"                                                
- host ===> Host from which traps are recieved ===> default value is "0.0.0.0" (listens on all interfaces)                                                    
- port ===> Port (optional) ===> default set to 162                                                                                                           
- community ===> Trap Community String which is used by the trap receiver to determine which traps are accepted from a device (Optional),                     
-                by default all devices has community string set to "public" by default, also useful to avoid unwanted floods of traps                        
-                from a malicious source ===> default value set is "pubic"                                                                                    
- trap_format ===> format of the output string                          
+ snmptrapalert input plugin for fluentd. This plugin is developed specifically to work on all versions of SNMP traps.
+ Plugin is specifically designed for snmp traps that are generated by devices. With this plugin installed,
+ SNMP traps that are generated by devices can be recieved by fluentd listener. The traps recieved by fluentd are formatted into a json output.
+
+ @type ===> type of input plugin (required) ===> 'snmptrapalert'
+ tag ===> tag that needs to be attached in front of each alert recieved ==> default value is "SNMPTrap.Alert"
+ host ===> Host from which traps are recieved ===> default value is "0.0.0.0" (listens on all interfaces)
+ port ===> Port (optional) ===> default set to 162
+ community ===> Trap Community String which is used by the trap receiver to determine which traps are accepted from a device (Optional),
+                by default all devices has community string set to "public" by default, also useful to avoid unwanted floods of traps
+                from a malicious source ===> default value set is "pubic"
+ trap_format ===> format of the output string
+
+## Testing
+```
+$ docker run -it --rm -v $PWD:/code ubuntu:16.04 bash
+$ apt-get update && apt-get install -y --no-install-recommends ruby-dev git gcc make libc6-dev smitools &&
+$ cd code &&
+$ gem install bundler:1.14 &&
+$ bundle install --path vendor/bundle &&
+$ bundle exec rake test
 
```

Comment:
end the code block here. Otherwise, the rest of the README gets formatted incorrectly